content/news/2023.pml

```diff
@@ -54,6 +54,31 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="microkit">
+  <a href="#microkit"><span class="tag-span"># </span></a>
+  27 October 2023: seL4 Microkit officially supported by the seL4 Foundation
+</div>
+<div class="news-text-container">
+  <div id="floated-imgs">
+    <a href="https://brkawy.com/">
+      <img src="/Foundation/Membership/LOGOS/Brkawy.png" style="width: 200px; padding:20px; float: right" alt="Breakaway Consulting logo">
+    </a>
+    <div class="style:clear:both"></div>
+    <a href="https://unsw.edu.au/">
+    <img src="/Foundation/Membership/LOGOS/UNSW.svg" style="width: 200px; padding:20px; float: right" alt="UNSW logo">
+    </a>
+  </div>
+  <p>
+     The seL4 Microkit, formerly known as the Core Platform, is an operating system framework on top of seL4 provides a small set of simple abstractions that ease the design and implementation of statically structured systems on seL4, while still leveraging the kernel’s benefits of security and performance. The Microkit is distributed as an SDK that integrates with the developer’s build system of choice, significantly reducing the barrier to entry for new users of seL4.
```

Comment:
We should really have a link here: https://docs.sel4.systems/projects/microkit/

Reply:
Also: typo: "provides" -> "that provides"

Reply:
@june-andronick Just waiting on the microkit logo. I've pencilled the release for 8 Nov.